app/mixins/google-pageview.js

```diff
@@ -1,13 +1,14 @@
 import Ember from 'ember';
 
 export default Ember.Mixin.create({
+
+    metrics: Ember.inject.service(),
+
     notifyGoogleAnalytics: Ember.on('didTransition', function() {
-        if (!window.ga) {
-            return;
-        }
-        return window.ga('send', 'pageview', {
-            page: this.get('url'),
-            title: this.get('url')
+        Ember.run.scheduleOnce('afterRender', this, () => {
+            const page = this.get('url');
+            const title = this.getWithDefault('currentRouteName', 'unknown');
+            Ember.get(this, 'metrics').trackPage({ page, title });
```

Comment:
just noticed that it's mentioned in the ember-metrics README like this, so it's probably fine. still wondering if it's really still needed though.